src/components/sidecard/Sidecard.css

```diff
@@ -0,0 +1,149 @@
+.bcgov-pos-relative {
+  position: relative;
+}
+
+.bcgov-heading-style {
+  font-weight: 600;
+  font-style: normal;
+  font-size: 16px;
+  color: #ffffff;
+  text-align: left;
+}
+
+.bcgov-grey-custom {
+  background-color: #F2F2F2 !important;
+  color: #000 !important;
+  border: none !important;
+}
+
+.bcgov-color-heading {
+  color: #000 !important;
+}
+
+.bcgov-bluegrey-heading {
+  padding-top: 10px;
+  padding-left: 15px;
+  padding-bottom: 9px;
+}
+
+.bcgov-bluegrey-heading h2 {
+  margin-bottom: 0px;
+}
+
+.bcgov-bluegrey-content {
+  height: auto;
+  background: inherit;
+  background-color: #f2f2f2;
+  box-sizing: border-box;
+  padding-top: 15px;
+}
+
+.bcgov-content-style {
+  padding-right: 15px;
+  padding-bottom: 10px;
+  padding-left: 15px;
+  display: flex;
+  flex-direction: column;
+}
+
+.bcgov-container-background {
+  background-color: rgba(56, 89, 138, 1);
+  font-weight: 400;
+  font-style: normal;
+  font-size: 12px;
+  text-align: left;
+}
+
+.bcgov-submitted-header h2 {
+  color: #fff !important;
+}
+
+.bcgov-submitted-content {
+  background-color: #eee;
+  padding: 30px;
+  margin-bottom: 20px;
+}
+
+.bcgov-submit-container {
+  background-color: #38598a;
+  border-bottom: solid 4px #fc0;
+  color: #fff;
+  margin-bottom: 20px;
+  padding: 30px;
+}
+
+.bcgov-bluegrey-container {
+  margin-bottom: 20px;
+}
+
+.bcgov-submit-container h2 {
+  color: #fff;
+}
+
+.bcgov-dashboard-spacing {
+  padding-left: 25px;
+  max-width: 400px;
+}
+
+.bcgov-wide-dashboard-spacing {
+  padding-left: 25px;
+  max-width: 458px;
+}
+
+img {
+  max-width: 100%;
+}
+
+@media (max-width: 575.98px) {
+  .bcgov-dashboard-spacing {
+    padding-left: 10px;
+    padding-right: 10px;
+  }
+
+  .bcgov-wide-dashboard-spacing {
+    padding-left: 0;
+  }
+}
+
+.bcgov-side-card-row {
+  white-space: nowrap;
```

Comment:
nit: 'nowrap' => should it be 'no-wrap' or camel case?

Reply:
![Screen Shot 2020-08-31 at 2 54 26 PM](https://user-images.githubusercontent.com/55710226/91772537-e30c9e00-eb99-11ea-85b6-1c8e916f1408.png)

Reply:
once again preset

Reply:
Thank  you. Learnt something!